app/helpers/helpers.py

```diff
@@ -0,0 +1,36 @@
+from flask import abort, make_response
+import requests
+from app import token
```

Comment:
Instead of importing `token` you can import `os` and set the value in this file where it is used. 

```python
import os

API_TOKEN = os.environ.get("API_TOKEN")